.github/workflows/.lychee.excludes

```diff
@@ -0,0 +1,2 @@
+file:///github/workspace/*
+file:///home/runner/work/opensearch-py/opensearch-py/docs/source/README.md
```

Comment:
These are pattern matched/regexes, I think it should work without `file://`, try `*/github/workspace/*` `*/docs/source/README.md`

Reply:
Basically removing sym link before link checker is my solution . Excluding (file:///home/runner/work/opensearch-py/opensearch-py/docs/source/README.md ) is needed . Let me give u a example, if u have [readme](docs/source/readme.md) as a relative link in some other doc, link checker will fail, because we removed sym link before link checking started . So to avoid this case I excluded that reference from link checker.so basically it's not about excluding file to be checked in link checker, it's about excluding the reference to be link checked.

Reply:
I tried excluding file using --exclude-path and --exclude-file , but it's excluding both readme files. So removing sym link may be the only option. At the end of link checking , I added sym link again so that it won't effect other workflow.